### ISSUE TITLE
Add label to search form

### ISSUE THREAD

**Issue by wheresrhys** [Financial-Times/o-header#140]
for accessibility reasons, but should be hidden by styles

**Comment by AlbertoElias** [comment]
what about using aria-label, the docs in mdn seem to consider this a valid use case?

**Comment by onishiweb** [comment]
I'll take a look into this today as there's an interesting article I've been meaning to read on the Aria labelled-by and label attributes

**Comment by onishiweb** [comment]
Ok, so having read: http://blog.jantrid.net/2015/12/woe-aria-surprisingly-but-ridiculously.html#sweet-and-this-applies-to-other-form-controls-too-yeah- 

Which says: 

```
If the label is visible, it’s easy: we render the label element and ignore the name of the control. If it isn’t visible, currently, NVDA browse mode doesn’t present it at all.
To solve this, we need to present the label separately.
```

So, I think the best approach will be to add a label element and make it visibly hidden but not from screenreaders, and then to test on a screenreader.

**Comment by onishiweb** [comment]
Looking into this further and testing with a screenreader, it turns out that a placeholder is enough to give it a "name" which shows up in Chrome (a11y tools) and is read out (albeit twice thanks to a Chrome/VO bug) using VoiceOver. 

![screen shot 2016-04-21 at 17 07 35](https://cloud.githubusercontent.com/assets/1240073/14716267/20c4287e-07e4-11e6-9509-ab9472ef7c26.png)
